src/components/Calculator.js

```diff
@@ -1,33 +1,13 @@
-import React from 'react';
-import '../App.css';
+/* eslint-disable */
+import React from "react";
+import "../App.css";
+import Calculate from "./Calculate.js";
 
 const Calculator = () => (
-  <div className="main">
-    <div className="calculator">
-      <div className="screen">0</div>
-      <div className="input-container">
-        <div className="inputs">AC</div>
-        <div className="inputs">+/-</div>
-        <div className="inputs">%</div>
-        <div className="inputs-right">/</div>
-        <div className="inputs">7</div>
-        <div className="inputs">8</div>
-        <div className="inputs">9</div>
-        <div className="inputs-right">x</div>
-        <div className="inputs">4</div>
-        <div className="inputs">5</div>
-        <div className="inputs">6</div>
-        <div className="inputs-right">-</div>
-        <div className="inputs">1</div>
-        <div className="inputs">2</div>
-        <div className="inputs">3</div>
-        <div className="inputs-right">+</div>
-        <div className="zero-input">0</div>
-        <div className="dot-inputs">.</div>
-        <div className="equal-inputs">=</div>
-      </div>
+    <div className="main">
+        <Calculate />
     </div>
-  </div>
 );
 
```

Comment:
- [ ] Great work but i guess you didn't understand what the previous reviewer asked 😄 Actually, it would be great if you make a child component of a Button or anything which you are using in the Calculator component more than once so you can use that child component in the Calculator Component which will serve the reusability and you will be able to understand how you can pass props from parents to child components 👍

**NOTE: You can see this [file](https://github.com/YoutLone/math-magicians/pull/2/commits/b9b168469e59129e197f37fccf3c93d6a52a1bc5#diff-2e862d0b88bdd42eb13f6528bc2ef48a56500daca0350696b337c7f2127f4f0b) to understand what i am talking about 😊**